### ISSUE TITLE
Unable to login through Google and Facebook on iOS 10 and iPhone 5c

### ISSUE THREAD

**Issue by eeschiavo** [firebase/FirebaseUI-iOS#565]
### Environment

  * Objective C or Swift: Swift
  * iOS version: 10.3.3.
  * Firebase SDK version: 5.9.0
  * FirebaseUI version: 5.2.2
  * CocoaPods Version: 1.5.3
  
### The problem:

I built a simple application following the [official tutorial](https://firebase.google.com/docs/auth/ios/firebaseui). The app does nothing special, just authenticate. It's working correctly both on simulator (iOS 12) and iPhone XS (iOS 12) but on iPhone 5c it doesn't seem to work: when I try to login through Google it opens the Google homepage instead of authenticate. Please check out [this video](https://www.dropbox.com/s/otbe7vib5t9oyni/Video%2009-11-18%2C%2021%2051%2036.mov?dl=0) I just made in order to quickly understand the problem. Please note that in Xcode console nothing appears.

At the same time also Facebook login is not working. When I tap on Facebook button I see this in console: 

> -canOpenURL: failed for URL: "fbauth2:/" - error: "This app is not allowed to query for scheme fbauth2"

then it lets me authenticate throughout Facebook but at the end nothing happens, it remain in the last step on Facebook website. I added an URL Type as explained in the official guide which consist of a string starting with "fb" and followed by the project id on Facebook.

I would like to highlight that the same app and exactly the same code is working correctly both on simulator and iPhone XS.
  
#### Relevant Code:

  ```

let providers: [FUIAuthProvider] = [FUIGoogleAuth(), FUIFacebookAuth()]
var authUI : FUIAuth?

///Here I set FIRAuth
func setFIRAuth() {
        
        self.authUI = FUIAuth.defaultAuthUI()
        self.authUI!.delegate = self
        self.authUI!.providers = providers
        
        Auth.auth().addStateDidChangeListener { (auth, user) in
            if user != nil {
                print("\(String(describing: user?.email))")                

                self.userLoggedIn = true
               
            } else {
                
                self.userLoggedIn = false
                self.presentFUILogin()
            }

        }
    }

   func presentFUILogin() {
        let authViewController = self.authUI!.authViewController()
        self.present(authViewController, animated: true, completion: nil)
    }
  ```

**Comment by eeschiavo** [comment]
I'm sorry I'm not sure to understand the question. The video is actually from my iPhone5c attached to Xcode (sorry for my English)

**Comment by morganchen12** [comment]
There's a sample app included in this repository in the `samples/swift` directory. Can you try reproducing this issue on your device with those sample projects?

**Comment by eeschiavo** [comment]
Sorry I didn’t understand. I’ll try for sure tomorrow and I will let you know! Thanks

**Comment by nowperson** [comment]
I have same issue, any idea?

**Comment by morganchen12** [comment]
Are either of you able to login with the sample app?

**Comment by IchordeDionysos** [comment]
Same issue and also tested with the sample project.
This doesn't work there.

Additional details on test:
I followed this steps:
git clone https://github.com/firebase/FirebaseUI-iOS.git
cd FirebaseUI-iOS/samples/swift
pod install
open FirebaseUI-demo-swift.xcworkspace

Which threw me an error at `pod install`

```
Fetching podspec for `FirebaseUI` from `../../`
[!] CocoaPods could not find compatible versions for pod "Firebase/Auth":
  In snapshot (Podfile.lock):
    Firebase/Auth (= 5.16.0, ~> 5.0)

  In Podfile:
    FirebaseUI (from `../../`) was resolved to 6.0.0, which depends on
      FirebaseUI/Auth (= 6.0.0) was resolved to 6.0.0, which depends on
        Firebase/Auth (~> 5.0)

It seems like you've changed the constraints of dependency `Firebase/Auth` inside your development pod `FirebaseUI`.
You should run `pod update Firebase/Auth` to apply changes you've made.
```

So I deleted the Podfile.lock and installed all dependencies at their newest versions.
With that I only added my app-id to the URL scheme and the Info.plist file,
turned the Keychain capability on 
and added the GoogleService-Info.plist file.

These should be the only steps to get Facebook Login runnig in the sample project right?

**Comment by IchordeDionysos** [comment]
Can we reopen the issue as this is critical to us?

**Comment by IchordeDionysos** [comment]
@all You can fix this by downgrading to `FirebaseUI 5.2.2` and add these lines 
```
pod 'FBSDKLoginKit', '4.38.1'
pod 'FBSDKCoreKit', '4.38.1'
```
to your Podfile because you have to downgrade `FBSDKLoginKit` and `FBSDKCoreKit 4.38.1`.

My Podfile looks like this:

```
  pod 'FirebaseUI/Auth', '5.2.2'
  pod 'FirebaseUI/Google', '5.2.2'
  pod 'FirebaseUI/Facebook', '5.2.2'
  pod 'FirebaseUI/Twitter', '5.2.2'
  pod 'FBSDKLoginKit', '4.38.1'
  pod 'FBSDKCoreKit', '4.38.1'
```

**Comment by IchordeDionysos** [comment]
I applied the fix suggested in this StackOverflow post: https://stackoverflow.com/questions/49124804/ios-facebook-login-sdk-4-28-0-error-code-3
